RecursiveExtractor.Blazor/Shared/DragnDropInputCommon.razor

```diff
@@ -28,7 +28,20 @@
         cursor: pointer;
     }
 </style>
+<h2>Encryption</h2>
+<input type="text" id="passwords" @bind="PasswordInput"/> <label for="password">Add a password to use</label> <button @onclick="AddPasswordToList">Add</button>
```

Comment:
Should this field type be password to mask/hide the value from people behind the user

Reply:
I’ll add a checkbox to hide password.